deployment/local-jvm/bash-scripts/attemptBulkTest.sh

```diff
@@ -1,26 +1,4 @@
 #!/usr/bin/env bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-. "$DIR/setScriptPath.sh"
-
-# Path for compiled JAR
-TARGET_DIR="${EXAMPLE}/example-model/target"
-
-FILE_PRESENT=0
-
-if [ -d "$TARGET_DIR" ];
```

Comment:
Why remove checks about whether the jar has been built and if not display a useful error?